# Ejercicios

#### 1 Ejercicio

Para calcular el importe de IVA de  145 Euros es necesario multiplicar 145 por 0.21.
Escribe en Python la expresión que calcula el IVA de 145 Euros.

In [6]:
# Sol:
x = 145
IVA_145 = x * 0.21
IVA_145

# El resultado es 30.45

30.45

Escribe en Python la expresión que calcula el IVA de 258 Euros.

In [10]:
# Sol:
x = 258
IVA_258 = x * 0.21
IVA_258

54.18

Escribe una función en Python llamada __calcula_iva__ que calcule el importe de IVA de una cierta cantidad $x$.

In [24]:
# Sol

def calcula_iva(x):
    iva = x * 0.21
    return iva

In [25]:
calcula_iva(9999)

2099.79

Ejecuta la función __calcula_Iva__ para las cantidades 145 y 258. Comprueba que los resultados son 30.45 y 54.18

#### 2 Ejercicio

Evalúa el polinomio $x^4 + x^3 + \frac{1}{2} x^2 - x$
en $x = 10$. El resultado
ha de ser  un número real.

Crear la expresion

In [29]:
# Sol:
def evalua(x):
    resultado = x**4 + x**3 + 1/2 * x**2 - x
    return resultado
float(evalua(10))

# El resultado es 11040.0

11040.0

In [30]:
evalua(7)

2761.5

Evalúa  el polinomio anterior para $x = 7$.

In [32]:
# Sol:
float(evalua(7))

2761.5

In [33]:
int(evalua(7))

2761

#### 3 Ejercicio

Define una función llamada __raiz_cubica__ que devuelva el valor de $\sqrt[3]{x}$. Recuerda que  $\sqrt[3]{x}$ = ${x}^{\frac{1}{3}}$


In [36]:
# Sol:
def raiz_cubica(x):
    resultado = x**(1/3)
    return resultado
float(raiz_cubica(5))

# El resultado de raiz cúbica de 5 es: 1.7099759466766968   

1.7099759466766968

In [40]:
int(raiz_cubica(5))

1

In [16]:
raiz_cubica(5)

1.7099759466766968

Escribe la expresión Python que construya la siguiente cadena:

$$"El \ resultado \ de  \  la  \  raíz  \  cúbica  \   de  \ 5 \  es:  1.70"$$

Utiliza el operador de formato __%f__.

In [39]:
'El %s de la %s cúbica de %d es: %.2f' % ('resultado', 'raíz', 5, 1.70)

'El resultado de la raíz cúbica de 5 es: 1.70'

--------------------